packages/subgraph/schema.graphql

```diff
@@ -1260,6 +1266,86 @@ type AccountTokenSnapshot @entity {
     account: Account!
     token: Token!
     flowOperators: [FlowOperator!]! @derivedFrom(field: "accountTokenSnapshot")
+    accountTokenSnapshotLogs: [AccountTokenSnapshotLog!]! @derivedFrom(field: "accountTokenSnapshot")
+}
+
+"""
+AccountTokenSnapshotLog: Historical entries of AccountTokenSnapshot updates.
+"""
+type AccountTokenSnapshotLog @entity {
+    id: ID!
+    timestamp: BigInt!
+    blockNumber: BigInt!
+
+    transactionHash: Bytes!
+    logIndex: BigInt!
+    order: BigInt!
+    triggeredByEventName: String!
```

Comment:
@0xdavinchee Could we use `triggeredByEvent: Event!` here?

Reply:
sure, but it may be useful to have both properties, one for filtering and the other for the information

Reply:
I guess... although in this case, the event ID actually contains all the same information, except the order, but of course it's not as explicit.

By the way, it seems that the Graph team is quite close to implementing filtering by child entity: https://github.com/graphprotocol/graph-node/pull/3184

Once that gets merged, we can have a more normalized schema, i.e. less duplicated fields just for filtering.